test/test_c10d.py

```diff
@@ -77,6 +77,37 @@ def simple_reduce_tests(rank, world_size):
     ]
 
 
+def simple_coalesced_reduce_tests(rank, world_size):
+    return [
+        (
+            c10d.ReduceOp.SUM,
+            [torch.Tensor([rank + 1]), torch.Tensor([(rank + 1) ** 2])],
```

Comment:
just a note: new code shouldn't use `torch.Tensor`, it should use `torch.tensor`.

Reply:
is `torch.Tensor` deprecated for good, or what's the difference between `torch.Tensor` and `torch.tensor` now?